### PR TITLE
reduce the amount of spelling issues / WORDLIST

### PR DESCRIPTION
**Pull request by m7pr**
A follow-up after https://github.com/insightsengineering/coredev-tasks/issues/235

I am basically trying to catch all spelling issues, and try to put \code{} around parameter names, function names, package names, object names and data structure names.

> `inst/WORDLIST` was reduced from 233 words to 71 words

Final result of `spelling::spell_check_package()`

```{R}
  WORD               FOUND IN
amongst            formatters_methods.Rd:186
                   gen_args.Rd:68
                   insert_row_at_path.Rd:20
                   int_methods.Rd:1192
                   label_at_path.Rd:17
                   make_afun.Rd:60
                   sort_at_path.Rd:21
                   ttap.Rd:17
Arg                compat_args.Rd:5,34
                   constr_args.Rd:5,124
                   lyt_args.Rd:5,208
                   sf_args.Rd:5,23
Carreras           README.md:199
                   README.Rmd:146
charset            build_table.Rd:41
                   gen_args.Rd:92
                   rtable.Rd:27
                   tabclasses.Rd:108
                   NEWS.md:113
Chohan             README.md:199
                   README.Rmd:146
combinatorial      split_functions.Rmd:147
CRAN's             NEWS.md:182
customizations     split_functions.Rmd:257
decrementing       paginate.Rd:172
dimensioned        head_tail.Rd:41
dplyr              tabulation_dplyr.Rmd:2
emph               custom_appearance.Rmd:108,248
facetted           qtable.Rd:58
facetting          qtable.Rd:27,29,51
formatter          CellValue.Rd:20
                   compat_args.Rd:18
                   format_rcell.Rd:21
                   rcell.Rd:35
                   rheader.Rd:15
                   rrow.Rd:16
                   rrowl.Rd:16
                   rtable.Rd:20
getter             col_accessors.Rd:123
                   NEWS.md:105
getters            formatters_methods.Rd:200
Godwin             README.md:199
                   README.Rmd:146
Heng               README.md:199
                   README.Rmd:146
ing                NEWS.md:92,107
initializer        custom_appearance.Rmd:116,170,176
Kelkhoff           NEWS.md:327
layouting          add_colcounts.Rd:18
                   add_existing_table.Rd:22
                   add_overall_col.Rd:17
                   analyze.Rd:73
                   analyze_colvars.Rd:49
                   append_topleft.Rd:16
                   basic_table.Rd:40
                   constr_args.Rd:103
                   split_cols_by.Rd:59
                   split_cols_by_multivar.Rd:44
                   split_rows_by.Rd:79
                   split_rows_by_multivar.Rd:69
                   summarize_row_groups.Rd:52
                   tabclasses.Rd:113
                   varcuts.Rd:149
                   NEWS.md:76
                   clinical_trials.Rmd:686
                   introduction.Rmd:145
                   sorting_pruning.Rmd:486
Layouting          lyt_args.Rd:5,208
                   README.md:221
                   README.Rmd:161
Lewandowski        README.md:199
                   README.Rmd:146
mandatorily        NEWS.md:115,205
Maximo             README.md:199
                   README.Rmd:146
Modelling          example_analysis_coxreg.Rmd:2
multivariable      example_analysis_coxreg.Rmd:16
NSE                clinical_trials.Rmd:396
orderable          sorting_pruning.Rmd:199
Paszty             README.md:199
                   README.Rmd:146
pathing            add_combo_facet.Rd:13
                   avarspl.Rd:92
                   cutsplits.Rd:84
                   formatters_methods.Rd:211,216,227
                   lyt_args.Rd:70,101
                   MultiVarSplit.Rd:40,59
                   split_cols_by_multivar.Rd:28
                   VarLevelSplit.Rd:82
                   NEWS.md:220
                   sorting_pruning.Rmd:232,256,438
                   subsetting_tables.Rmd:195
Phuse              clinical_trials.Rmd:435
postprocessing     add_combo_facet.Rd:6,27
                   split_functions.Rmd:273
Postprocessing     make_split_fun.Rd:35,67
                   trim_levels_in_facets.Rd:5,18
pre                add_colcounts.Rd:10
                   add_existing_table.Rd:10
                   add_overall_col.Rd:10
                   analyze.Rd:24
                   analyze_colvars.Rd:18
                   append_topleft.Rd:10
                   build_table.Rd:19,51
                   custom_split_funs.Rd:25,28,98
                   int_methods.Rd:1096
                   lyt_args.Rd:55
                   make_split_fun.Rd:73
                   matrix_form-VTableTree-method.Rd:18
                   split_cols_by.Rd:21,78,81
                   split_cols_by_multivar.Rd:18
                   split_funcs.Rd:83,86
                   split_rows_by.Rd:25,105,108
                   split_rows_by_multivar.Rd:23
                   summarize_row_groups.Rd:19
                   varcuts.Rd:89
                   vil.Rd:11,34
                   NEWS.md:78
                   custom_appearance.Rmd:108
                   example_analysis_coxreg.Rmd:373
                   introduction.Rmd:138,409
                   sorting_pruning.Rmd:386
                   split_functions.Rmd:46,281,290,300,309,411
Pre                col_accessors.Rd:119
                   example_analysis_coxreg.Rmd:25
                   split_functions.Rmd:431,434,442
priori             split_functions.Rmd:151
programmatically   README.md:62
                   README.Rmd:56
Qi                 README.md:199
                   README.Rmd:146
quartiles          varcuts.Rd:152,159
reindexed          head_tail.Rd:62
repo               README.md:18,29
                   README.Rmd:29,35
repped             analyze.Rd:85
                   analyze_colvars.Rd:21
                   names.Rd:29
responder          clinical_trials.Rmd:740
Resync             NEWS.md:165
reusability        clinical_trials.Rmd:401
RinPharma          README.md:210
                   README.Rmd:153
roadmap            clinical_trials.Rmd:889
Saibah             README.md:199
                   README.Rmd:146
sortable           sorting_pruning.Rmd:438
spl                analyze.Rd:145
                   spl_context.Rd:5,7,9
                   spl_context_to_disp_path.Rd:5,17
Stoilova           README.md:199
                   README.Rmd:146
subtable           add_existing_table.Rd:16
                   analyze.Rd:63
                   analyze_colvars.Rd:39
                   avarspl.Rd:100
                   CellValue.Rd:32
                   cutsplits.Rd:97
                   int_methods.Rd:1125
                   label_at_path.Rd:27,29,31
                   lyt_args.Rd:160
                   ManualSplit.Rd:37
                   MultiVarSplit.Rd:72
                   paginate.Rd:45
                   rcell.Rd:45
                   rowclasses.Rd:60
                   split_rows_by.Rd:63
                   split_rows_by_multivar.Rd:54
                   summarize_row_groups.Rd:41
                   tabclasses.Rd:92
                   table_structure.Rd:16
                   trim_prune_funs.Rd:59,63
                   validate_table_struct.Rd:18
                   varcuts.Rd:143
                   VarLevelSplit.Rd:95
                   NEWS.md:116
                   introduction.Rmd:332
                   sorting_pruning.Rmd:30,111,119,137,199,222,232,237,290,296,344
Subtable           sorting_pruning.Rmd:288
subtable's         sorting_pruning.Rmd:207
subtables          basic_table.Rd:18,20
                   constr_args.Rd:79,81
                   find_degen_struct.Rd:13
                   horizontal_sep.Rd:29
                   paginate.Rd:97,138
                   qtable.Rd:39,41
                   sort_at_path.Rd:80
                   tabclasses.Rd:96,98
                   validate_table_struct.Rd:17
                   NEWS.md:39,114
                   introduction.Rmd:359,368
                   sorting_pruning.Rmd:28,31,102,111,191,207,237,461
Subtables          trim_prune_funs.Rd:36
summarization      tabulation_dplyr.Rmd:128,162
tableone           introduction.Rmd:36
Tadeusz            README.md:199
                   README.Rmd:146
unicode            build_table.Rd:40
                   gen_args.Rd:91
                   rtable.Rd:26
                   tabclasses.Rd:107
univariable        example_analysis_coxreg.Rmd:16
unnested           df_to_tt.Rd:13
unpruned           sorting_pruning.Rmd:147
unstratified       clinical_trials.Rmd:745
Unstratified       clinical_trials.Rmd:712
useR               README.md:224
                   README.Rmd:163
                   introduction.Rmd:55
xtable             introduction.Rmd:35
Yung               README.md:199
                   README.Rmd:146
```

### CODE REVIEW

**Comment by shajoezhu** [comment]
hi @m7pr , many thanks for this, can I suggest we do the sweep right before https://github.com/insightsengineering/rtables/issues/678? there are a few PRs are about to be merged in

**Comment by m7pr** [comment]
Hey @shajoezhu - I am ready with my part. I suggest we merge now, so everything that is required by #678 will already have spell-check fixed and you will only need to figure out the new spelling issues that will be triggered by GitHub Actions spelling check. I reckon it will be easier to solve future spelling issues one-by-one on a single PR than once again once multiple PRs are merged.

**Comment by Melkiades** [comment]
Thanks, Marcin to work on this. I think we need to put it on hold merging it after  https://github.com/insightsengineering/rtables/pull/665. What you think @shajoezhu @gmbecker

File: vignettes/introduction.Rmd
Comment (m7pr) [inline vignettes/introduction.Rmd:32]:
```suggestion

```

File: tests/testthat/test-lyt-tabulation.R
Comment (m7pr) [inline tests/testthat/test-lyt-tabulation.R:362]:
```suggestion
                 "Error.*custom split function: oopsie daisy.*VarLevelSplit \\(RACE\\).*path: ARM\\[A: Drug X\\]")
```

File: tests/testthat/test-lyt-tabulation.R
Comment (m7pr) [inline tests/testthat/test-lyt-tabulation.R:352]:
```suggestion
        stop("oopsie daisy")
```

**Comment by Melkiades** [comment]
> IMHO the sooner you merge, the sooner you start paying attention to spelling anywhere

Trust me, we did know very well about the problem which predates the current SME team.

Anyway, if you insist we can merge it in already, but I have the feeling that this is going to break all PRs currently open and it is not a great timing for this. 

In general, we need to collaborate on this, and at least warn us about it. Just let us know before starting working on these kinds of issues as it may not be the best timing for those. Also, this work is not on Kendis, and it is clearly something that has dependencies on our team.

**Comment by Melkiades** [comment]
> > IMHO the sooner you merge, the sooner you start paying attention to spelling anywhere
> 
> Trust me, we did know very well about the problem which predates the current SME team.
> 
> Anyway, if you insist we can merge it in already, but I have the feeling that this is going to break all PRs currently open and it is not a great timing for this.

But this is my council: flag this on @gogonzo to liaison with @shajoezhu on the best way forward before any merge is being considered. In the interim, hold fire on this one.

**Comment by pawelru** [comment]
> In general, we need to collaborate, and please at least warn us about it beforehand

Apologies guys. The whole blame is on me. Bad planning. I chatted with Marcin and I ask him to complete that for rtables repo. Anyway, @shajoezhu please make a call what to do with this PR. Even if it will wait - I don't expect a lot of complex git conflicts to be resolved.

**Comment by shajoezhu** [comment]
hey guys, thanks for working on this. I am blocking the merge for now

**Comment by m7pr** [comment]
Apologies, I was not aware that there is release going on. Take your time to merge it. Flag me also when you think it can be merged and I'll fix conflicts and update files after you made changes to your PRs.

**Review by shajoezhu — CHANGES_REQUESTED** [review]
hi @m7pr , can you add a NEWS entry please. Thanks!

**Comment by m7pr** [comment]
Yes @shajoezhu - I did fix conflicts and updated NEWS with a relevant note about the spelling revision

**Review by shajoezhu — APPROVED** [review]
lgtm! many thanks @m7pr

**Pull request merged**
